worker.js

```diff
@@ -71,13 +71,18 @@ async function handleDELETE(request) {
  */
 async function handleRequest(request) {
 	const url = new URL(request.url);
-	const path = url.pathname.split('/')[1];
+	const path = url.pathname.split('/')[1];	
 	if (!path) {
 		// Return list of available shortlinks if user supplies admin credentials.
 		const psk = request.headers.get('x-preshared-key');
 		if (psk === SECRET_KEY) {
-			const { keys } = await LINKS.list();
-			return new Response(JSON.stringify(keys), { status: 200 });
+			const {keys} = (await LINKS.list());
+			let links = "";
```

Comment:
I think the variable name `paths` might be more appropriate because the response is a list of paths and not links per se